docs/getting-started-provider-dev.md

```diff
@@ -216,6 +216,8 @@ export EXTERNAL_CLOUD_PROVIDER_BINARY=$GOPATH/src/k8s.io/cloud-provider-openstac
 export EXTERNAL_CLOUD_PROVIDER=true
 export CLOUD_PROVIDER=openstack
 export CLOUD_CONFIG=/etc/kubernetes/cloud-config
+export HOSTNAME_OVERRIDE=$(hostname)
```

Comment:
@ramineni I think as of now, we should not add this to docs, We can file an issue with the error you are getting if not exporting HOSTNAME_OVERRIDE.

Reply:
ok, I will do that

Reply:
Done.